WEB/Src/PerformanceCollector/PerformanceCollector/QuickPulseTelemetryProcessor.cs

```diff
@@ -126,6 +129,18 @@ public void Initialize(TelemetryConfiguration configuration)
             this.RegisterSelfWithQuickPulseTelemetryModule();
         }
 
+        void IQuickPulseTelemetryProcessor.UpdateGlobalQuotas(Clock timeProvider, float? maxGlobalTelemetryQuota, float? initialGlobalTelemetryQuota, float? quotaAccrualRatePerSec)
+        {
+            if (maxGlobalTelemetryQuota != null || initialGlobalTelemetryQuota != null || quotaAccrualRatePerSec != null)
```

Comment:
So if all of them are _null_, we just silently don't do anything? Wouldn't that be very confusing?